examples/failover/main.tf

```diff
@@ -0,0 +1,79 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ROUTE53 ZONE WITH RECORDS THAT HAVE A FAILOVER ROUTING POLICY ATTACHED
+#
+# Failover routing lets you route traffic to a resource when the resource is healthy or to a different resource when the
+# first resource is unhealthy. The primary and secondary records can route traffic to anything from an Amazon S3 bucket
+# that is configured as a website to a complex tree of records.
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Configure the AWS Provider
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "aws" {
+  region = var.aws_region
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# We configure two records with associated healthchecks. Route53 will route the traffic to the secondary record if
+# the healthcheck of the primary record reports an unhealthy status.
+#
+# For more information on DNS failover with Route53, please check the related documentation:
+# https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy.html#routing-policy-failover
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_route53_health_check" "primary" {
```

Comment:
The tests are failing as expected. We need to address the issue with computed values to get this working @mariux 
<img width="923" alt="Screen Shot 2020-02-18 at 16 37 49" src="https://user-images.githubusercontent.com/930643/74751264-49b91280-526d-11ea-8960-605c1b34c130.png">